projects/schematics/src/add-spartacus/schema.json

```diff
@@ -155,6 +156,10 @@
             "value": "EPD-Visualization",
             "label": "EPD Visualization Integration"
           },
+          {
+            "value": "OPF-Checkout",
+            "label": "Open Payment Framework Checkout Integration"
+          },
```

Comment:
[QUESTION]

What option(s) do we want to show in x-prompt for OPF feature(s), in the end of the day?

Here I can see only an option to install OPF-Checkout (which implicitly will make sure to install OPF-Base, see [opf-schematics-config.ts#L73](https://github.com/SAP/spartacus/blob/e3b1abd4b857384143b2c8165746d426985d2a9e/projects/schematics/src/shared/lib-configs/integration-libs/opf-schematics-config.ts#L73)).

But I cannot see an option to install only OPF-Base. Do we want to add it to the x-prompt opptions, so customers can install just the OPF-Base without OPF-Checkout??

Or maybe totally different: maybe we want expose just a single general x-prompt option named "OPF" - which will install both feature modules - opf base and opf checkout?

In the near future, do we plan to introduce more optional parts of the OPF epic?

IMHO the decision here should driven by the business requirements (usual customers' habits) - what parts of OPF epic customers may want to use exclusively (in parts) vs. whether they always want to use them all together?

Reply:
[OUTDATED] We decided to install all OPF sub-libs by installing a base. It is not choosable.